### PR TITLE
add i18n: zh-TW

### PR DESCRIPTION
**Pull request by tonytonyjan**

### CODE REVIEW

**Comment by graysonwright** [comment]
谢谢！

Can anyone from the community verify?

**Comment by tonytonyjan** [comment]
@graysonwright  Actually, "谢谢" is zh-CN. In zh-TW, it's "謝謝" :smile: 

I guess there are few Taiwanese people knowing this project now, but maybe @JuanitoFates can help, he is the core contributor of translating Rails Guide to zh-TW.

And, I once talked the similar topic about how to build a good admin in Ruby conference Taiwan few years ago. It follows the same rule: "let developers write Rails code they’re already familiar with". I really like your gem, thanks for building this awesome project :+1:

**Comment by graysonwright** [comment]
@tonytonyjan ah, thought I might have messed that up. thanks!

I'd love to see the talk - do you have a link that I can check out?

**Comment by tonytonyjan** [comment]
@graysonwright Unfotunately, [my talk](http://tonytonyjan.net/slides/2014-04-25-yet-another-rails-admin) is in Traditional Chinse, there was simultaneous interpreting then, but I have no permission to get the media file, sorry :(

File: config/locales/administrate.zh-TW.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+zh-TW:
+  administrate:
+    actions:
+      confirm: 你確定？
```
Comment (JuanitoFatas) [inline config/locales/administrate.zh-TW.yml]: `你` seems a little redundant here.

File: config/locales/administrate.zh-TW.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    controller:
+      create:
+        success: "已成功新增 %{resource}。"
+      destroy:
+        success: "已成功刪除 %{resource}。"
+      update:
+        success: "已成功更新 %{resource}。"
+    fields:
+      has_many:
+        more: 顯示 %{total_count} 筆中的 %{count} 筆資料
```
Comment (JuanitoFatas) [inline config/locales/administrate.zh-TW.yml]: I haven't tried administrate yet, could you point me where this translation being used from the demo site or a screenshot? :blush:

Comment (graysonwright) [inline config/locales/administrate.zh-TW.yml]: At this URL it's showing up under the "Line Items" table: https://administrate-prototype.herokuapp.com/admin/orders/291

order__291___administrate_prototype

Comment (JuanitoFatas) [inline config/locales/administrate.zh-TW.yml]: @graysonwright Thanks!

:+1:

**Comment by JuanitoFatas** [comment]
LGTM

**Comment by graysonwright** [comment]
Thanks! Merged in as https://github.com/thoughtbot/administrate/commit/cef797bddf01617ac03670875d58df493e0a9ceb

**Comment by JuanitoFatas** [comment]
@georgebrock awesome!

**Comment by graysonwright** [comment]
(=

**Pull request closed without merging**